### PR TITLE
Unit Test Coverage: italic-controller.ts #483

### PR DESCRIPTION
**Pull request by rsknutson**
Issue #483

### CODE REVIEW

**Comment by bharney0** [comment]
@rsknutson Thank you for your contribution to Docs Authoring Pack. We'll be looking into the unit test coverage and let you know if we have feedback.

File: docs-markdown/src/test/suite/controllers/italic-controller.test.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        formatItalic();
+        await sleep(300);
+        expect(spy).to.have.been.called();
+        stub.restore();
+    });
+    test("Italic Format Multiple Selection", async () => {
+        const filePath = resolve(__dirname, "../../../../../src/test/data/repo/articles/docs-markdown.md");
+        await loadDocumentAndGetItReady(filePath);
+        const editor = window.activeTextEditor;
+        common.setSelectorPosition(editor!, 159, 0, 159, 4);
```
Comment (bharney0) [inline docs-markdown/src/test/suite/controllers/italic-controller.test.ts:73]: Can you use `editory.selections` so we are doing multiple selections. Currently this just selects one selection. But we want to test multiple selections. See here https://code.visualstudio.com/api/references/vscode-api#TextEditor `Selection[]`. You should be able to do this by doing 
```
editor.selections = [
    new Selection(fromPosition, toPosition), 
    new Selection(secondFromPosition, secondToPosition)
]
```

Comment (rsknutson) [reply docs-markdown/src/test/suite/controllers/italic-controller.test.ts:73]: Requested usage of Selection is now implemented

Comment (bharney0) [reply docs-markdown/src/test/suite/controllers/italic-controller.test.ts:73]: looks good, thanks for the updated changes.

**Pull request merged**